src/main/java/de/hsrm/vegetables/my_food_coop_service/controller/ReportsController.java

```diff
@@ -41,6 +44,15 @@
     @NonNull
     private UserService userService;
 
+    @NonNull
+    private DisposedRepository disposedRepository;
```

Comment:
Do not use repositories in controllers. A repository is solely controller by a service - the gateway to the database.